### ISSUE TITLE
Reaction picker should stay open after selecting a reaction

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#11266]
Possibly with the tooltip semantics from before? It's annoying to have to click so much to add multiple reactions.

**Comment by nadonomy** [comment]
I'm not sure I agree with this suggestion— we should prioritise the UX for the common case, which in this instance is adding a single reaction.

**Comment by Wendell250** [comment]
What about holding “Shift” to add multiple reactions?
Same as Discord and Slack.

**Comment by turt2live** [comment]
I no longer believe in this issue and would like to take my name off of it. If folks feel strongly otherwise, please open a new issue for reconsideration. 

ftr: Nad convinced me way back in 2019 but clearly I forgot to communicate that 😅
